test/integration2/query/insert/on-conflict-deferred.spec.js

```diff
@@ -0,0 +1,76 @@
+const {
+  getAllDbs,
+  getKnexForDb,
+} = require('../../util/knex-instance-provider');
+const { isPostgreSQL } = require('../../../util/db-helpers');
+
+describe('Insert', () => {
+  describe('onConflict deferred', () => {
+    getAllDbs().forEach((db) => {
+      describe(db, () => {
+        let knex;
+        before(() => {
+          knex = getKnexForDb(db);
+          if (!isPostgreSQL(knex)) {
+            this.skip('This test is PostgreSQL only');
```

Comment:
this.skip is not available in arrow function, you'd need to change it to `function()`

Reply:
Thank you, I learned something new today. :)